### PR TITLE
Bugfix/scrolling boundries

### PR DESCRIPTION
**Pull request by vuongp**
Improve imagin when using viewpager2.

see: https://issuetracker.google.com/issues/123006042

### CODE REVIEW

File: library/src/main/java/nl/nos/imagin/ScrollHandler.kt [hunk 1]
```diff
@@ -59,9 +64,22 @@ class ScrollHandler(
                 }
             })
 
+    private fun ImageView.leftEdgeIsVisible(): Boolean {
```
Comment (mennovogel) [inline library/src/main/java/nl/nos/imagin/ScrollHandler.kt:72]: Shouldn't this be `translationX == 0`?

File: example/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -18,7 +18,9 @@
             </intent-filter>
         </activity>
 
-        <activity android:name=".gallery.GalleryActivity" />
+        <activity android:name="nl.nos.imagin.example.gallery_vp1.GalleryActivity" />
```
Comment (mennovogel) [inline example/src/main/AndroidManifest.xml]: Why would you keep this GalleryActivity?

Comment (vuongp) [reply example/src/main/AndroidManifest.xml]: No reason! I'll revert this

File: library/src/main/java/nl/nos/imagin/ScrollHandler.kt (hunk 1 above)
Comment (vuongp) [reply library/src/main/java/nl/nos/imagin/ScrollHandler.kt:72]: That is only true if the image is not scaled. If the image is scaled(zoomed in) than translation==0 would be the image in the center cropped.

**Review by mennovogel — APPROVED** [review]

**Review by TimGW — APPROVED** [review]

**Pull request merged**